src/app/(sidebar)/endpoints/components/SavedEndpointsPage.tsx

```diff
@@ -186,7 +210,130 @@ export const SavedEndpointsPage = () => {
   };
 
   const RpcEndpoints = () => {
-    return <Card>Coming soon</Card>;
+    if (savedRpcMethods.length === 0) {
+      return <Card>There are no saved RPC Methods</Card>;
+    }
+
+    return (
+      <Card>
+        <Box gap="md">
+          {savedRpcMethods.map((e, idx) => (
+            <Box
+              gap="sm"
+              key={`horizon-${e.timestamp}`}
+              addlClassName="PageBody__content"
+            >
+              <Box gap="sm" direction="row">
+                <Badge size="md" variant="secondary">
+                  {e.rpcMethod}
+                </Badge>
+              </Box>
+              <div className="Endpoints__urlBar">
+                <Input
+                  id={`endpoint-url-${e.timestamp}`}
+                  fieldSize="md"
+                  value={e.url}
+                  readOnly
+                  leftElement={
+                    <InputSideElement
+                      variant="text"
+                      placement="left"
+                      addlClassName="Endpoints__urlBar__requestMethod"
+                    >
+                      {e.method}
+                    </InputSideElement>
+                  }
+                />
+              </div>
+              <Box
+                gap="lg"
+                direction="row"
+                align="center"
+                justify="space-between"
+                addlClassName="Endpoints__urlBar__footer"
+              >
+                <Box gap="sm" direction="row">
```

Comment:
We'll need to add a `wrap` prop to this `Box` to wrap buttons if they don't fit on smaller screens. This prop was added in [my branch](https://github.com/stellar/laboratory/blob/sign-txn-updated-ux/src/components/layout/Box/index.tsx#L12), which is not merged yet.

Example: set it manually for the first item.

![image](https://github.com/user-attachments/assets/8049c311-2593-4dfe-95d6-7c0a6f0e1e3d)

Reply:
i promise i'll check mobile from now on